### PR TITLE
feat(Audiomack): Add presence

### PR DESCRIPTION
**Pull request by KannaDev**
**Describe the changes in this pull request:**
Added Now Playing for Audiomack, currently only shows the now playing song but working on adding the current progress in the song, if paused or resumed and when browsing :3
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<br>

image
</details>

### CODE REVIEW

**Comment by KannaDev** [comment]
Not sure exactly what the cause of the issue is, would be great if someone could help resolve the issue with the schema validation check :3

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/A/Audiomack/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	},
+	"url": "audiomack.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/c0y4XDa.png",
+	"thumbnail": "https://i.imgur.com/M3Ft6dL.png",
+	"color": "#ffa200",
+	"tags": ["audiomack", "music"],
+	"category": "music",
+	"iframe": false,
+	"readLogs": false
```
[the remark this answers is not in the document]
Comment (KannaDev) [reply websites/A/Audiomack/dist/metadata.json]: removed 👍🏻

File: websites/A/Audiomack/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Runa ٩(＾◡＾)۶",
```
Comment (KannaDev) [inline websites/A/Audiomack/dist/metadata.json:4]: Is it required to use our Discord name here? As I primarily go by Kanna and only using Runa as a temporary name.

Comment (Bas950) [reply websites/A/Audiomack/dist/metadata.json:4]: Feel free to use Kanna

Comment (KannaDev) [reply websites/A/Audiomack/dist/metadata.json:4]: Alright 👍🏻

**Comment by KannaDev** [comment]
Everything should be fine unless if you need me to make any further changes :3

**Comment by EncryptedDev** [comment]
Hello @KannaDev. 

Myself and the Team believe this should either be closed or held as a draft until further features are added. I understand you have addressed this originally, but we would rather not merge this pull request until it has more features added to it 
> working on adding the current progress in the song, if paused or resumed and when browsing

I will convert this to a draft, but you're more than welcome to close it if you cannot add the features in a reasonable time frame.

**Comment by EncryptedDev** [comment]
Hello @KannaDev, as I haven't seen any progression here, I will be closing the pull request. Feel free to reopen with the extra features. 

As always, contact me or another member of the Team if you need some help 👍🏼

**Pull request closed without merging**